.github/workflows/push.yml

```
@@ -72,3 +72,33 @@ jobs:
+
+      - name: Run Flake8 Linter
+        run: |
+          flake8 . --ignore E402,E722,E203,F401
+
+      - name: Run Black Formatter Check
+        run: |
+          black --check .
+
+      - name: Run Pydocstyle for Docstring Style
+        run: |
+          pydocstyle switchmap setup bin tests --convention=google --add-ignore=D415,D205
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)